app/cli/internal/action/workflow_list.go

```diff
@@ -28,14 +28,16 @@ type WorkflowList struct {
 }
 
 type WorkflowItem struct {
-	Name       string           `json:"name"`
-	ID         string           `json:"id"`
-	Team       string           `json:"team"`
-	Project    string           `json:"project,omitempty"`
-	CreatedAt  *time.Time       `json:"createdAt"`
-	RunsCount  int32            `json:"runsCount"`
-	ContractID string           `json:"contractID,omitempty"`
-	LastRun    *WorkflowRunItem `json:"lastRun,omitempty"`
+	Name            string           `json:"name"`
+	ID              string           `json:"id"`
+	Team            string           `json:"team"`
+	Project         string           `json:"project,omitempty"`
+	CreatedAt       *time.Time       `json:"createdAt"`
+	RunsCount       int32            `json:"runsCount"`
+	ContractID      string           `json:"contractID,omitempty"`
+	LastRun         *WorkflowRunItem `json:"lastRun,omitempty"`
+	RobotAccountID  string           `json:"robotAccountID,omitempty"`
```

Comment:
I am not sure this is the best place to put robot account properties. 

This struct is in some way a translation between the server object and the client one, and there is an issues with this approach now. A workflow returned by the server will never fill those two fields, in fact those fields are set by the client only on creation. 

What about rendering the message outside in the cmd/workflow_create.go instead? See my comment there

Reply:
OK. I was just trying to reuse as much as I can what we already have, especially that ContractID is there as well. The problem I see now is that all these fields are probably present in the workflow db table and RobotAccount does not really fit there because it is 1-to-many. I will change it.

Reply:
> OK. I was just trying to reuse as much as I can what we already have

Yes, that makes sense. But in fact my proposal requires less changes, you'll just need to just update one file (I think), `cmd/workflow_create.go`

Reply:
But your proposal does not allow me to automate it because in -o json, I will not be able to get the token. Any ideas?

Reply:
I see, in my head the scope of this was to improve the default user console experience.

Two options, either skipping the creation of the token if you are not in table mode, or if we think it's important to also support json output in one go, go back to your approach. Either way, you choose, i'll approve it.

Reply:
> Or you propose to return:
> 
> * a RA json object if we autocreate RA
> * and workflow json for the current behaviour?
> 
> I did not like this approach because this is confusing and not the best UX

No, my proposal would be to not to auto-create robot-account if you are not in `table` mode. Potentially showing a warning with the logger that the ra will not be created.

But your initial proposal is fine if we want to support everything, maybe adding some comments with the caveats of that output (only works on creation and does not represent the actual set of robot-accounts associated)

Reply:
> either skipping the creation of the token if you are not in table mode

This is important, because this is confusing. How will we explain it? We have more parameters: skip-ra-create(true or false) and output (json or table) -> this makes four different scenarios

Reply:
A "notice" would suffice imo. "robot account creation skipped during json output"

Reply:
Then I would prefer skip this for now and just ask the user to do `chainloop workflow robot-account create`. We are going to review the cli UX soon anyway. What do you think?